tests/dummy/app/templates/login.hbs

```diff
@@ -2,8 +2,15 @@
 <input
   type="text"
   value={{this.jwt}}
-  {{on "input" this.updateJwt}} />
-
-<button type="button" {{on "click" this.login}}>{{t "general.login"}}</button>
+  {{on "input" (pick "target.value" (set this.jwt))}}
+  {{on "keypress" (perform this.loginOnEnter)}}
+  disabled={{this.loginOnEnter.isRunning}}
```

Comment:
I feel like this should be disabled whenever `login` is running, not just on enter.

Reply:
you're right. i corrected this.